nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ScanAttribute.java

```diff
@@ -60,13 +64,21 @@
 @Tags({"scan", "attributes", "search", "lookup"})
 @CapabilityDescription("Scans the specified attributes of FlowFiles, checking to see if any of their values are "
         + "present within the specified dictionary of terms")
+@WritesAttributes({
+        @WritesAttribute(attribute = "dictionary.hit.{n}.attribute", description = "The attribute name that had a value hit on the dictionary file."),
+        @WritesAttribute(attribute = "dictionary.hit.{n}.term", description = "The term that had a hit on the dictionary file."),
+        @WritesAttribute(attribute = "dictionary.hit.{n}.metadata", description = "The metadata returned from the dictionary file associated with the term hit.")
+})
+
+
 public class ScanAttribute extends AbstractProcessor {
 
     public static final String MATCH_CRITERIA_ALL = "All Must Match";
     public static final String MATCH_CRITERIA_ANY = "At Least 1 Must Match";
 
     public static final PropertyDescriptor MATCHING_CRITERIA = new PropertyDescriptor.Builder()
-            .name("Match Criteria")
```

Comment:
Must retain the previously existing name so as not to disturb existing configurations.  Andy's advice for the new property is good to follow going forward but it is wise to avoid changing old names.